ga/latest/kernel/helpers/build/configure.sh

```diff
@@ -12,13 +12,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Determine if featureUtility ran in an earlier build step
+if /opt/ibm/helpers/build/features-installed.sh; then
+  FEATURES_INSTALLED=true
+else
+  FEATURES_INSTALLED=false
+fi
+
 if [ "$VERBOSE" != "true" ]; then
   exec &>/dev/null
 fi
 
 set -Eeox pipefail
 
 function main() {
+  if [ "$FEATURES_INSTALLED" == "false" ]; then
+    # Resolve liberty server symlinks and creation for server name changes
```

Comment:
could this block be skipped for now and be added later when the custom server name is supported?

Reply:
removed the block